### ISSUE TITLE
Creating new seminars, projects, etc

### ISSUE THREAD

**Issue by mkgold** [cuny-academic-commons/openlab-theme#191]
As I browse the CBOX-OL as a logged in user, when I go to some of the established group types (Seminars / Projects / Clubs) and view all existing ones of that type, I do not see an option to create a new [seminar/project/club). Am I right that the only place to start the creation of a new [seminar/project/club] is from one's own profile, or am I missing something?

**Comment by boonebgorges** [comment]
Yes, this is the only way to launch the creation process.

**Comment by bree-z** [comment]
See also #121 .  I think that adding a group creation link on the group directory page is a good idea.  And I believe this was something on @charliemedwards 's list as well.

**Comment by mkgold** [comment]
Boone, is it too late to add this? I would say that it would be good to have a "Create [Group type]" on every [Group Type] directory page if possible. That is how we have set up the CAC -- e.g. https://commons.gc.cuny.edu/groups/ and https://commons.gc.cuny.edu/sites/

**Comment by boonebgorges** [comment]
I'd suggest it's too late. We would need IA design, as well as comps that take mobile + desktop view into account. The technical lift is not large but it's likely we don't have the team bandwidth.

**Comment by charliemedwards** [comment]
Hi there - another vote for adding this, if at all possible. I'm pretty sure we said this was something we were going to do... :)

**Comment by jennaspevack** [comment]
Maybe something like this on every group page? It's not really ideal, because the original design wasn't really set up up for it, but.... 

Boone, I think the text would be 14px, weight 600. Hover would be underline.
![createclone-pages](https://user-images.githubusercontent.com/22797200/40257290-b1952412-5abb-11e8-8244-becd0be705b8.png)

**Comment by boonebgorges** [comment]
Thanks, @jennaspevack. Can you give some guidance about what to do on narrower viewports? I think the breakpoint is 766px or so. Below that, the binoculars appear:

![screenshot_2018-05-18_16-49-10](https://user-images.githubusercontent.com/246627/40259401-8cf57e90-5abb-11e8-949b-abe97d140f0d.png)

I'm going to have to totally rebuild the markup and styling for these directory headers to accommodate the new link, so I want to make sure I've accounted for the various permutations.

**Comment by jennaspevack** [comment]
Sure thing, sorry missed that. I'd suggest hiding the link on mobile. There is too much stuff cramped up there at the top on mobile. The h2 breadcrumbs (from Search/Filter), which are not shown in your screen shot, are misaligned and wrap a bit weirdly right now (OL issue), but even if those are sorted out, it's just too messy in my opinion -- and not absolutely necessary, since the link can be found on Group Home. Also, the link would look strange just hanging out alone, if there are no groups to show. Not sure if that's okay with Matt et al, but it would require a bit more design thinking to get this sorted out and not sure if there is time.

**Comment by boonebgorges** [comment]
Thanks, Jenna. Leaving out seems like a prudent way forward for now. I'll see what needs to be done to implement on desktop.

**Comment by mkgold** [comment]
Huge thanks for doing this, Boone -- I think it's an important feature and your changes look great. Many apologies for not raising this earlier in the development process, but thank you for implementing.

**Comment by boonebgorges** [comment]
Thanks. I think this can be tested then.

**Comment by boonebgorges** [comment]
@bree-z Just be sure you've had a look at each type of directory (group types, members), in both mobile and desktop view, to be sure that nothing's amiss. Thanks, and sorry for the extra work :(

**Comment by jennaspevack** [comment]
I agree with Bree about hiding the link on the directory page if the user already has a portfolio - if that is possible. If not, then I'd say just remove it. It's confusing, because it takes you to your Profile page.

**Comment by boonebgorges** [comment]
Sounds good. I've added the logic so that you only see the link on the Portfolio directory if you don't have a portfolio.
